package.json

```diff
@@ -111,6 +111,7 @@
     "ps-node": "0.1.6",
     "request": "2.88.2",
     "rimraf": "3.0.2",
+    "s3-access-log-parser": "0.0.2",
```

Comment:
I prefer to avoid this dependancy. The parsing should not be that complicated, and I don't like the idea of adding a dependency just for that (security issues, maintainability, etc.). 
looking at the [github repo](https://github.com/icodeforlove/s3-access-log-parser), it looks like it is not maintained in the last 6 years